swagger_to/elm_client.py

```diff
@@ -600,7 +590,12 @@ def write_request(request: Request, fid: TextIO) -> None:
     types_str = ' -> '.join(types)
     types_str += ' -> '
     names_str = ' '.join(names)
-    line1 = function_name + ' : ' + types_str + 'Http.Request {}\n'.format(return_type)
+
+    line1 = function_name + ' : ' + types_str
+    if return_type:
```

Comment:
Please add a comparison operator here -- it makes the code a bit more robust (`is not None` or `return_type == ''`?).

Though this is considered unpythonic, I really find it difficult to distinguish the two cases and already experienced bugs due to it.

It's probably best to make:

```python3
return_type = None  # type: Optional[str]
...
```